### ISSUE TITLE
docker-compose not found in boot2docker

### ISSUE THREAD

**Issue by zhangqianliang** [docker/compose#1032]
I install docker-compose on boot2docker:

curl -L https://github.com/docker/compose/releases/download/1.1.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
chmod +x /usr/local/bin/docker-compose

But:
docker@boot2docker:~$ which docker-compose
/usr/local/bin/docker-compose
docker@boot2docker:~$ docker-compose --version
-sh: docker-compose: not found

**Comment by bfirsh** [comment]
Unfortunately Compose doesn't run on boot2docker at the moment. You'll have to run it on the host computer and point it at the boot2docker VM, as explained in the docs.

See https://github.com/docker/compose/issues/599 for more information.

Sorry about that!

**Comment by ganlanyuan** [comment]
Hey,
I'm running **docker for mac** (1.12.1-rc1).
And my **Dockerfile** is
```
FROM node

RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E9C74FEEA2098A6E && \
    echo "deb http://packages.dotdeb.org/ wheezy-php55 all" > /etc/apt/sources.list.d/php.list && \
  apt-get update && \
  apt-get install -y php5-cli php5-curl php5-fpm php5-gd php5-mcrypt php5-mysql php5-pgsql php5-sqlite
ADD php-fpm.conf /etc/php5/fpm/php-fpm.conf

VOLUME ['/www/web']
WORKDIR /www/

# Global install gulp
RUN npm set progress=false && \
    npm install --global --progress=false gulp bower && \
    echo '{ "allow_root": true }' > /root/.bowerrc

COPY package.json /www/package.json
RUN npm install

EXPOSE 8000 3000 3001

COPY docker-start.sh /www/docker-start.sh
```
**docker-compose.yml**
```
version: '2'
services:
  web:
    image: web
    ports:
      - 8000:8000
      - 3000:3000
      - 3001:3001
    volumes:
      - .:/www/web/
    entrypoint: /www/docker-start.sh
    container_name: web
    tty: true
```
When I run `docker-compose up -d`, I get the error **no such file or directory**
```
WARNING: The Docker Engine you're using is running in swarm mode.

Compose does not use swarm mode to deploy services to multiple nodes in a swarm. All containers will be scheduled on the current node.

To deploy your application across the swarm, use the bundle feature of the Docker experimental build.

More info:
https://docs.docker.com/compose/bundles

Starting web

ERROR: for web  Cannot start service web: oci runtime error: exec: "/www/docker-start.sh": stat /www/docker-start.sh: no such file or directory
ERROR: Encountered errors while bringing up the project.
```
Could someone help me with this issue?
Thanks

**Comment by ismarslomic** [comment]
You can install docker-compose in boot2docker by following the official installation guideline: https://docs.docker.com/compose/install/

**Comment by ganlanyuan** [comment]
Hey, thanks
My issue has been fixed #3876

**Comment by v6** [comment]
// , I got this problem after following the instructions here: 

https://docs.docker.com/engine/installation/linux/ubuntu/#install-using-the-repository
